osbenchmark/metrics.py

```diff
@@ -224,7 +224,10 @@ class IndexTemplateProvider:
     """
 
     def __init__(self, cfg):
-        self.script_dir = cfg.opts("node", "benchmark.root")
+        self._config = cfg
+        self.script_dir = self._config.opts("node", "benchmark.root")
+        self._number_of_shards = self._config.opts("reporting", "datastore.number_of_shards", default_value=None, mandatory=False)
+        self._number_of_replicas = self._config.opts("reporting", "datastore.number_of_replicas", default_value=None, mandatory=False)
```

Comment:
This is good -- it is exactly what I thought would be needed to resolve the issue.